.github/workflows/tests.yml

```
@@ -0,0 +1,57 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', 'pypy-3.7-nightly']
```

That version of PyPy is no longer supported.
```suggestion
        python-version: ['3.8', '3.9', '3.10', 'pypy-3.11-nightly']
```

---

I didn't update this old file and I will try to fix that.